packages/sidenav/src/ux-sidenav-drawer/ux-sidenav-drawer.ts

```diff
@@ -0,0 +1,41 @@
+import { useView, customElement, PLATFORM, inject, bindable } from "aurelia-framework";
+import { normalizeBooleanAttribute } from "@aurelia-ux/core";
+import { UxDefaultSidenavConfiguration } from "../ux-default-sidenav-configuration";
+
+export const OPEN_CHANGED = 'open-changed';
```

Comment:
Can this be either `OPEN_CHANGED_EVENT` on the class, or in an shared exported enum